### PR TITLE
Automate the rules doc pushes to github pages

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -40,11 +40,20 @@ module.exports = function(grunt) {
     'webpack:build',
   ]);
 
-  grunt.registerTask('publish-rules', [
-    'copy',
-    'buildrules',
-    'gh-pages',
-  ]);
+  grunt.registerTask('publish-rules',
+                      'Used by travis to publish rule docs', function() {
```
Comment (tofumatt) [inline Gruntfile.js]: That's an odd indentation.

File: docs/rules.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 
 # Validator Rules
 
-We should try and list out all the validation rules we know of by file type. E.g. .js, .rdf, .manifest etc.
+This is automatically published at http://mozilla.github.io/addons-validator/.
+To update the rules update docs/rules.md in the
```
Comment (tofumatt) [inline docs/rules.md]: I like surrounding filenames with \` so it's `docs/rules.md`.

**Comment by tofumatt** [comment]
r+wc, cool :+1:

**Comment by muffinresearch** [comment]
Looks like the addons-robot doesn't have the rights to push yet. I'm going to sort that by getting it added to a specific team.

**Pull request merged**